src/uwtools/config/tools.py

```diff
@@ -170,38 +160,82 @@ def _print_config_section(config: dict, key_path: List[str]) -> None:
     print("\n".join(sorted(output_lines)))
 
 
+def _realize_config_input_setup(
+    input_config: Union[Config, Optional[Path]] = None, input_format: Optional[str] = None
+) -> Tuple[Config, str]:
+    """
+    Set up config-realize input.
+
+    :param input_config: Input config source (None => read stdin).
+    :param input_format: Format of the input config.
+    :return: The input Config object and its format name.
+    """
+    input_format = _ensure_format("input", input_format, input_config)
+    if not input_config:
+        log.debug("Reading input from stdin")
```

Comment:
This is an example of new debug logging showing users, when they specify `--verbose` (or connect `uwtools` to a verbose logger via the API), what is being read from `stdin`.